ddsrouter_core/test/blackbox/ddsrouter_core/dds/WAN/DDSTestWAN.cpp

```diff
@@ -49,41 +192,44 @@ void test_WAN_communication(
     uint32_t samples_sent = 0;
     std::atomic<uint32_t> samples_received(0);
 
+    // Create a message with size specified by repeating the same string
     HelloWorld msg;
-    msg.message("Testing DDS-Router Blackbox WAN...");
+    std::string msg_str;
+
+    // Add this string as many times as the msg size requires
+    for (uint32_t i = 0; i < msg_size; i++)
+    {
+        msg_str += "Testing DDSRouter Blackbox Local Communication ...";
+    }
+    msg.message(msg_str);
 
     // Create DDS Publisher in domain 0
-    HelloWorldPublisher<HelloWorld> publisher;
+    DummyDDSPublisher<HelloWorld> publisher;
```

Comment:
When I read `DummyDDSPublisher` I thought this was referring to the `Dummy` Participant. I'd avoid using the `Dummy` key word here. I think `TestPublisher` could be more adequate.